.eslintrc.yml

```diff
@@ -222,8 +222,7 @@ rules:
     - error
     - max: 1
   multiline-ternary:
-    - error
-    - never
+    - off
```

Comment:
If you agree, of course :) Ternary operations are very handy, but the lines can get long very quickly,

Reply:
Wait, does this enable the use of multiline ternaries? Hell yes! 👍